govcd/catalog.go

```diff
@@ -202,58 +223,94 @@ func (cat *Catalog) UploadOvf(ovaFileName, itemName, description string, chunkSi
 	for _, filePath := range filesAbsPaths {
```

Comment:
We are assuming here that the extracted file contains a .ovf.
If this is not true, the procedure hangs. This check needs to happen before the item is created.